money-monkey-web/app/lib/api.ts

```
@@ -30,38 +31,50 @@ export async function getCategories(): Promise<Category[]> {
+export async function getCategorySums(
+  month: number,
+  year: number
+): Promise<CategorySum[]> {
+  var userId = auth().userId;
   if (userId == null) {
-    throw new Error("could not find userId")
+    throw new Error("could not find userId");
   }
 
-  var url = `${process.env.API_BASE}/categories/sums?year=${year}`
+  var url = `${process.env.API_BASE}/categories/sums?year=${year}`;
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)